entries/async.xml

```diff
@@ -41,7 +46,30 @@ QUnit.test( "two async calls", function( assert ) {
 });
 ]]></code>
   </example>
+    <example height="250">
+  <desc>Establish an accept call count. The <code>done()</code> callback can be called at accpet call count.</desc>
```

Comment:
How about "Set up an async test three exit points. Each <code>done()</code> call adds up to the <code>acceptCallCount</code>. After three calls, the test is done."